fabric-networking-api-v1/src/main/java/net/legacyfabric/fabric/api/networking/v1/EntityTrackingEvents.java

```diff
@@ -14,22 +15,23 @@
  * limitations under the License.
  */
 
-package net.fabricmc.fabric.api.networking.v1;
+package net.legacyfabric.fabric.api.networking.v1;
+
+import net.legacyfabric.fabric.api.event.Event;
+import net.legacyfabric.fabric.api.event.EventFactory;
 
 import net.minecraft.entity.Entity;
+import net.minecraft.server.network.EntityTrackerEntry;
 import net.minecraft.server.network.ServerPlayerEntity;
 
-import net.fabricmc.fabric.api.event.Event;
-import net.fabricmc.fabric.api.event.EventFactory;
-
 /**
  * Events related to a tracking entities within a player's view distance.
  */
 public final class EntityTrackingEvents {
 	/**
 	 * An event that is called before player starts tracking an entity.
 	 * Typically this occurs when an entity enters a client's view distance.
-	 * This event is called before the player's client is sent the entity's {@link Entity#createSpawnPacket() spawn packet}.
+	 * This event is called before the player's client is sent the entity's {@link EntityTrackerEntry#method_33558() spawn packet}.
```

Comment:
someone should map this

Reply:
noted

Reply:
Mapped this, @BoogieMonster1O1 make the change